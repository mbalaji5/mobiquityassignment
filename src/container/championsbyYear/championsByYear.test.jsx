import React from "react";
import { render } from "@testing-library/react";
import ChampionsByYear from "./championsbyYear";

describe("Should validate champion by year", () => {
  it("Should validate shopshot", () => {
    const { baseElement } = render(<ChampionsByYear />);
    expect(baseElement).toMatchSnapshot();
  });
});
