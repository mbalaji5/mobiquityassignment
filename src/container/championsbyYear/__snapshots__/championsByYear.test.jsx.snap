// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render champion detail shopshot Should validate shopshot 1`] = `
<body>
  <div>
    <div
      class="championsByYear"
    >
      <div>
        ...Loading
      </div>
    </div>
  </div>
</body>
`;

exports[`Should validate champion by year Should validate shopshot 1`] = `
<body>
  <div>
    <div
      class="championsByYear"
    >
      <div>
        ...Loading
      </div>
    </div>
  </div>
</body>
`;

exports[`Should validate champion by year Should validate shopshot 2`] = `
<body>
  <div>
    <div
      class="championsByYear"
    >
      <div>
        ...Loading
      </div>
    </div>
  </div>
</body>
`;
